src/main/resources/io/sealights/plugins/sealightsjenkins/buildsteps/cli/CommandMode/EndView/config.jelly

```diff
@@ -4,10 +4,12 @@
     <f:entry title="Build Session Id" field="buildSessionId">
         <f:textbox value="${buildSessionId}" default="$${SL_BUILD_SESSION_ID}"/>
     </f:entry>
-
+    <f:entry title="Lab Id" field="labId">
```

Comment:
Does it include the EndExecution in the post build step.

Reply:
yes